mungegithub/issue-labeler/simple_app.py

```diff
@@ -0,0 +1,125 @@
+import os
+import simplejson
+import logging
+from logging.handlers import RotatingFileHandler
+
+import numpy as np
+from flask import Flask, request
+from sklearn.feature_extraction import FeatureHasher
+from sklearn.externals import joblib
+from sklearn.linear_model import SGDClassifier
+from nltk.tokenize import RegexpTokenizer
+from nltk.stem.porter import PorterStemmer
+app = Flask(__name__)
+
+#Parameters
+team_fn= "./models/trained_teams_model.pkl"
+component_fn= "./models/trained_components_model.pkl"
+logFile = "/tmp/issue-labeler.log"
+logSize = 1024*1024*100
+numFeatures = 262144
+myLoss = 'hinge'
+myAlpha = .1
+myPenalty = 'l2'
+myHasher = FeatureHasher(input_type="string", n_features= numFeatures, non_negative=True)
+myStemmer = PorterStemmer()
+tokenizer = RegexpTokenizer(r'\w+')
+
+
+try:
+  stop_words = []
+  if not stopwords:
+    stop_fn = "./data/stopwords.txt"
```

Comment:
modified the dockerfile to add it to the container.  there are only 100ish words but I don't want to copy them all into the source file.